src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
 // write code here
+
+const table = document.querySelector('.field > tbody');
+const buttons = document.querySelector('.container');
+
+function modifyColumns(rezising) {
+  switch (rezising) {
+    case 'append-row':
+      table.appendChild(table.querySelector('tr').cloneNode(true));
+      break;
+    case 'remove-row':
+      table.deleteRow(table.rows.length - 1);
```

Comment:
Check if the table has any rows before attempting to delete one. If `table.rows.length` is zero, this will throw an error.